core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -393,7 +412,7 @@ private RaftClient createClient() {
         .setMaxSleepTime(TimeDuration.valueOf(mConf.getElectionTimeoutMs(), TimeUnit.MILLISECONDS))
         .build();
     return RaftClient.newBuilder()
-        .setRaftGroup(mRaftGroup)
+        .setRaftGroup(updateGroup ? getCurrentGroup() : mInitialRaftGroup)
```

Comment:
Why do we ever want to work with the old quorum info?

Reply:
Usually during the start up before a raft quorum is formed.